<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditScientist</class>
 <widget class="QDialog" name="EditScientist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; color:#000000;&quot;&gt;Edit scientist&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineNameScientist"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboGender">
     <item>
      <property name="text">
       <string>Female</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Male</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Year of birth:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLineEdit" name="lineYearOfBirth">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Year of death: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineYearOfDeath">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="placeholderText">
      <string>Skip if still alive</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonOkCancel">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>accepted()</signal>
   <receiver>EditScientist</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOkCancel</sender>
   <signal>rejected()</signal>
   <receiver>EditScientist</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
